src/stan/math/prim/scal/meta/VectorView.hpp

```diff
@@ -46,8 +46,8 @@ namespace stan {
 
     VectorView(std::vector<scalar_t>& v) : x_(&v[0]) { }
 
-    template <int R, int C>
-    VectorView(Eigen::Matrix<scalar_t,R,C>& m) : x_(&m(0)) { }
+    template <typename ET>
```

Comment:
I see. So the `Derived` type within the `MatrixBase< Derived >` is an actual `Eigen::Matrix<scalar_t, -1, -1>`, right? I'm not sure what's supposed to happen here, then.